libarchive/archive_read_support_format_tar.c

```diff
@@ -2298,11 +2345,10 @@ pax_attribute(struct archive_read *a, struct tar *tar, struct archive_entry *ent
 					return (err);
 				}
 				else if (key_length == 4 && memcmp(key, "size", 4) == 0) {
-					/* GNU.sparse.size */
+					/* GNU.sparse.size = size of stored entry */
```

Comment:
Ouch.  This comment isn't actually correct; `GNU.sparse.size` might be the size of the stored data or the size of the file on disk, depending on the GNU sparse format version.